server/server/src/static_files.rs

```diff
@@ -7,12 +7,18 @@ use actix_web::web::Data;
 use actix_web::{guard, web, Error, HttpRequest, HttpResponse};
 use reqwest::StatusCode;
 use serde::Deserialize;
+use service::plugin_files::PluginFileService;
 use service::settings::Settings;
 use service::static_files::StaticFileService;
 
 // this function could be located in different module
 pub fn config_static_files(cfg: &mut web::ServiceConfig) {
     cfg.service(web::resource("/files").guard(guard::Get()).to(files));
+    cfg.service(
```

Comment:
was allowing the fetching by just the plugin name (i.e. `/plugins/ShippingStatus` as well as individual files required by the plugin app such as `/plugins/ShippingStatus/ShippingStatus.js` and `/plugins/ShippingStatus.312.js` for a specific chunk )

the `/plugins/{plugin}` request will resolve to the file `/plugins/{plugin}/{plugin}.js` on the file system.. just as a helper really. The client could make the full request, I just thought this was neater

Reply:
yes - easy enough to do that. Just thought the default was neat, there's no practical reason for it

Reply:
> But wouldn't the frontend always figure out the filename and put it in the request? i.e. never call this endpoint without filename?

I agree